os_stub/mbedtlslib/include/mbedtls/config.h

```diff
@@ -1239,7 +1239,7 @@ extern void my_free( void *ptr );
  *
  * Uncomment this macro to disable the built-in platform entropy functions.
  */
-/*#define MBEDTLS_NO_PLATFORM_ENTROPY*/
+#define MBEDTLS_NO_PLATFORM_ENTROPY
```

Comment:
Please dont unconditional enable this MACRO.

Reply:
Thanks. I have added the judgement for this MACRO.